content/en/docs/collector/deployment/gateway.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Gateway
+description:
+  Why and how to send signals to a single OTLP end-point and from there to
+  backends
+weight: 3
+---
+
+The gateway collector deployment pattern consists of applications (or other
+collectors) sending telemetry signals to a single OTLP endpoint provided by one
+or more collector instances running as a standalone service (for example, a
+deployment in Kubernetes), typically per cluster, per data center or per region.
+
+In the general case you can use an out-of-the-box load balancer to distribute
+the load amongst the collectors:
+
+![Gateway deployment concept](../../img/gateway-sdk.svg)
+
+For use cases where the processing of the telemetry data processing has to
+happen in a specific collector, you would use a two-tiered setup with a
+collector that has a pipeline configured with the [Trace ID/Service-name aware
+load-balancing exporter][lb-exporter] in the first tier and the collectors
+handling the scale out in the second tier. For example, you will need to use the
+load-balancing exporter when using the [Tail Sampling
+processor][tailsample-processor] so that all spans for a given trace reach the
+same collector instance where the tail sampling policy is applied.
+
+Let's have a look at such a case where we are using the load-balancing exporter:
+
+![Gateway deployment with load-balancing exporter](../../img/gateway-lb-sdk.svg)
+
+1. In the app, the SDK is configured to send OTLP data to a central location.
+1. A collector configured using the load-balancing exporter that distributes
+   signals to a group of collectors.
+1. The collectors are configured to send telemetry data to one or more backends.
+
+> **Note** Currently, only trace pipelines are supported by the load-balancing
+> exporter.
```

Comment:
I think we should create those Notes/Warnings consistently

@chalin is this syntax to prefer over the hugo shortcodes? https://www.docsy.dev/docs/adding-content/shortcodes/#alert WDYT?

Reply:
Yes, we should use the `alert` shortcode when we can. I haven't been consistent in enforcing this yet simply because I was wanting to tweak the shortcode first. But it is good enough to be used as is for now. Thanks for flagging this.

Reply:
Addressed this, using the alert shortcode now